user/Lambda.py

```diff
@@ -67,12 +55,30 @@ def Lambda_Acquire(acquire_time=0.1, acquire_period=0.11,num_images=100,file_nam
     @bpp.stage_decorator([scaler1])
     @bpp.monitor_during_decorator(monitored_things)
     def inner():
+        # write metadata to the dm_pars (various registers)
+        yield from bps.mv(
+            dm_pars.root_folder, file_path,
+            dm_pars.parent_folder, os.path.dirname(file_path),
+            dm_pars.data_folder, file_name,
+            dm_pars.datafilename, areadet.get_plugin_file_name(),
+            # TODO: what else?
+        )
+
         md = {
             "file_name": file_name,
             "file_path": file_path
         }
-        yield from bps.mv(scaler1.count,"Count")
-        yield from bp.count([adlambda], md=md)
+        yield from bps.mv(scaler1.count, "Count")
+        yield from bp.count([areadet], md=md)
+
+        # FIXME: need correct filename here: hdf_with_fullpath
+        yield from dm_workflow.create_hdf5_file(hdf_with_fullpath, as_bluesky_plan=True)
+        
+        # kickoff the DM workflow
+        # TODO: which one?
+        # TODO: should run in thread, does it?
```

Comment:
these dm routines return information that should be logged

    id=1f131fd4-76e2-4e0f-b97f-d0acd809242b owner=8idiuser status=pending startTime=1567697398.43 startTimestamp=2019/09/05 10:29:58 CDT

Reply:
@sureshnaps Where should this be logged?  Along with the other logged output already generated?

Reply:
I put it in the logging for now.  We can change this.